src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -315,7 +315,7 @@ JAVADOC -> <audio><source src="horse.ogg" type="audio/ogg"/></audio><EOF> [0:0]
 |       |   `--CLOSE -> > [0:6]
 |       |--HTML_ELEMENT -> <source src="horse.ogg" type="audio/ogg"/> [0:7]
 |       |   `--SINGLETON_ELEMENT -> <source src="horse.ogg" type="audio/ogg"/> [0:7]
-|       |       `--SINGLETON_TAG -> <source src="horse.ogg" type="audio/ogg"/> [0:7]
+|       |       `--EMPTY_TAG -> <source src="horse.ogg" type="audio/ogg"/> [0:7]
```

Comment:
As I understand it, this issue is about replacing written paragraph text inside this file, not about changing the printed tree structure.
This tree item is a `SINGLETON_TAG`, so the change is incorrect.